modules/Bio/EnsEMBL/Compara/PipeConfig/EBI/Ensembl/PrepareMasterDatabaseForRelease_conf.pm

```diff
@@ -32,12 +32,12 @@ Bio::EnsEMBL::Compara::PipeConfig::EBI::Ensembl::PrepareMasterDatabaseForRelease
 
 =head1 DESCRIPTION
 
-    Add/update all species to master database
+    Prepare master database for next release
 
 
 =head1 SYNOPSIS
 
-    init_pipeline.pl Bio::EnsEMBL::Compara::PipeConfig::EBI::Ensembl::PrepareMasterDatabaseForRelease_conf
+    init_pipeline.pl Bio::EnsEMBL::Compara::PipeConfig::EBI::Ensembl::PrepareMasterDatabaseForRelease_conf  -release <release>
```

Comment:
I've pushed a change last week so that the parameter used is `ensembl_release`, not `release`. `ensembl_release` is set in eHive's `EnsemblGeneric_conf` and is used in all the other Compara pipelines